features/admin.core.v1/components/modals/feature-preview-modal.tsx

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Button from "@oxygen-ui/react/Button";
+import Card from "@oxygen-ui/react/Card";
+import Container from "@oxygen-ui/react/Container";
+import Dialog from "@oxygen-ui/react/Dialog";
+import DialogActions from "@oxygen-ui/react/DialogActions";
+import DialogTitle from "@oxygen-ui/react/DialogTitle";
+import FormControlLabel from "@oxygen-ui/react/FormControlLabel";
+import Grid from "@oxygen-ui/react/Grid";
+import List from "@oxygen-ui/react/List";
+import ListItem from "@oxygen-ui/react/ListItem";
+import ListItemButton from "@oxygen-ui/react/ListItemButton";
+import ListItemText from "@oxygen-ui/react/ListItemText";
+import Switch from "@oxygen-ui/react/Switch";
+import Typography from "@oxygen-ui/react/Typography";
+import {
+    ConnectorPropertyInterface,
+    ServerConfigurationsConstants,
+    updateGovernanceConnector,
+    useGetGovernanceConnectorById
+} from "@wso2is/admin.server-configurations.v1";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import React, { ChangeEvent, FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import NewSelfRegistrationImage from "../../assets/illustrations/preview-features/new-self-registration.png";
+
+/**
+ * Feature preview modal component props interface. {@link FeaturePreviewModal}
+ */
+interface FeaturePreviewModalPropsInterface extends IdentifiableComponentInterface {
+    open: boolean;
```

Comment:
Let's add doc comments for interface members

Reply:
https://github.com/wso2/identity-apps/pull/7855